gui/default/assets/lang/lang-en.json

```diff
@@ -59,6 +59,7 @@
    "Device ID": "Device ID",
    "Device Identification": "Device Identification",
    "Device Name": "Device Name",
+   "Device where the last modification was done": "Device where the last modification was done",
```

Comment:
@calmh How about this?

Reply:
I still find the two modification columns confusing and somehow out of place in the out of sync modal.  If you really think they add value for the user I would (not native speaker) suggest "last modified by this device", "last modifying device" or "device that lastly modified this file"

Reply:
Agree with Ben, except s/lastly/last/. I think it's the past perfect tense (?) in "was done" instead of something more direct like "did" that sounds more German than English to me (as a swede, mind). :)

Reply:
Well, that's again closer to my first proposal, which @calmh didn't like (and I agree in hindsight), because the device didn't modify anything, the user did...  
Also at least the which device last modified were useful in one scenario: One device that isn't in use anymore got some index data to one device which then show up as out of sync, Without the information about the device, I would have never known why they are even there. I agree this is a very remote scenario, but "debugging" usually involves weird scenarios which you didn't think of before, so I believe it is good to stay on the verbose side of things.

Reply:
Another suggestion: simply use "device" and "time" as table header - much easier for translators because I think not everybody gets the abbreviation if he does not see the code. Optional tooltips e.g. "device that last modified this file" - optional because I think we should only include them if we get a torrent of support requests regarding these columns.

If we leave both columns in place my only request would be to have "time" column before "device" column, thanks.

Reply:
@uok @calmh @AudriusButkevicius 
I am honestly slightly annoyed due to many different proposals but no clear consensus. I really don't mind much either way, I'd just like to get to a decision (which could also be do whatever you want, but that's also a decision). I will use the following unless there is clear opposition (as in I get overruled by authority/majority or my previous arguments are addressed and refuted :P ):

A. Should the "Mod." be dropped?  
I agree having the abbreviation is a bit ugly, "Modification" is just very long, so I considered it the lesser evil. I am against dropping this, as it introduces ambiguity (e.g. time when sync will finish/started, time created, ...). Yes if you actually think about it, such alternatives don't make much sense, but assuming users think upfront is a leap of faith.  
I will keep "Mod." as it is currently.

B. What tooltip to use:
**"Device that last modified this file"** is perfectly fine with me, I assume @calmh is now also ok with the slight discrepancy in the meaning between the device vs the user making the change.

@uok I will swap the time and device columns. I did not bring up dropping the columns alltogether as they were originally proposed by @AudriusButkevicius (which I blatantly interpret as approval) and I am for having them. And I don't understand the idea of dropping the the tooltips until there is confusion instead of preventing that upfront: What is the problems with tooltips (except finding consensus on their wording xD )?

Reply:
I only stumbled upon "mod." as a word when thinking about translations as I'm not sure all translators are programmers. Let's got with what you suggested, thanks.